index.js

```diff
@@ -5,7 +5,8 @@ module.exports = class TracingConfigPlugin {
      this.serverless = serverless;
      this.options = options;
      this.aws = serverless.getProvider('aws');
-     this.hooks = {
+
+     if(!options.noDeploy) this.hooks = {
```

Comment:
1. Could you please wrap the if statement body with the curly braces? 
```
if(...) {
  this.hooks = ...
}
```
2. What if `options.noDeploy` is a string `"false"`?
3. Please add a unit test in `test.js`.